assets/test.yaml

```diff
@@ -1,10 +1,9 @@
-input: /nfs/treeoflife-01/teams/tola/users/dp24/ascc/assets/samplesheet.csv
+input: /Users/dp24/Documents/ascc/assets/local_testing/samplesheet.csv
```

Comment:
path to local file

Reply:
assets/test.yaml isn't actually used at all. I instead use folders named: github_testing, farm_testing and production_testing to differentiate it all.

I'll delete test.yaml